scripts/configs/eval/eval_params.yaml

```diff
@@ -6,20 +6,19 @@ model_display_names: ["test_model"]
 
 fifty_one_dataset_name: "rick_data_test"
 
-# list with absolute paths to new unlabeled videos folders for prediction
+# str with an absolute path to a directory containing videos for prediction.
 # (it's not absolute just for the toy example)
-path_to_test_videos: ["toy_datasets/toymouseRunningData/unlabeled_videos"]
+test_videos_directory: "toy_datasets/toymouseRunningData/unlabeled_videos"
 
 # list with absolute paths to the .csv with predictions for each vid in path_to_test_videos
-path_to_csv_predictions:
-  ["toy_datasets/toymouseRunningData/unlabeled_videos/test_vid_heatmap_temporal_0.000000.csv"]
-
-# path to save the .csv file with preds
-path_to_save_predictions:
-  ["toy_datasets/toymouseRunningData/unlabeled_videos/"]
+saved_vid_preds_dir: "toy_datasets/toymouseRunningData"
 
 # params for loading images from the video using nvidia-dali
 dali_parameters:
-
   # how many frames to grab at once from the video
   sequence_length: 16
+
+video_file_to_plot: "/home/jovyan/lightning-pose/toy_datasets/toymouseRunningData/unlabeled_videos/test_vid.mp4"
```

Comment:
would be helpful to update the comment for each parameter in this file, users should ideally be able to read this file and understand exactly what each of these is for